packages/beacon-node/src/api/rest/swaggerUI.ts

```diff
@@ -0,0 +1,32 @@
+import {FastifyInstance} from "fastify";
+import {BeaconRestApiServerOpts} from "./index.js";
+
+export async function registerSwaggerUIRoutes(
+  server: FastifyInstance,
+  opts: BeaconRestApiServerOpts,
+  version = ""
+): Promise<void> {
+  await server.register(await import("@fastify/swagger"), {
+    openapi: {
+      info: {
+        title: "Lodestar API",
+        description: "",
+        version,
+        contact: {
+          name: "Lodestar Github",
+          url: "https://github.com/chainsafe/lodestar",
+        },
+      },
+      externalDocs: {
+        url: "https://chainsafe.github.io/lodestar",
+        description: "Lodestar documentation",
+      },
+      tags: opts.api.map((namespace) => ({name: namespace})),
+    },
+  });
+  await server.register(await import("@fastify/swagger-ui"), {
+    theme: {
+      title: "Lodestar API",
+    },
+  });
```

Comment:
I thought about adding that.
This would break in npm installs, since the assets are not published, but I guess we don't like npm installs anyways. Maybe we can do this in a way that can gracefully degrade.